.github/workflows/ios-ci.yml

```diff
@@ -0,0 +1,55 @@
+name: Vividus iOS App CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: NPM cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: macos-ios-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            macos-ios-node-
+      
+      - name: Cocoapods cache
+        uses: actions/cache@v2
+        with:
+          path: ios/Pods
+          key: macos-ios-pods-${{ hashFiles('**/Podfile.lock') }}
+          restore-keys: |
+            macos-ios-pods-
+
+      - name: Install npm dependencies
+        run: npm install
+      
+      - name: Run lint
+        run: npm run lint
+
+      - name: Run test
+        run: npm run test
+
+      - name: Select Xcode 11.3
+        run: sudo xcode-select --switch /Applications/Xcode_11.3.app
+
+      - name: Install bundler
+        run: |
+          sudo bundle update --bundler
+          sudo bundle install
+
+      - name: iOS build
+        env:
+          SAUCELABS_USER: ${{ secrets.SAUCELABS_USER }}
+          SAUCELABS_KEY: ${{ secrets.SAUCELABS_KEY }}
+        run: npm run build-ios
```

Comment:
only builds from `master` branch should be published to SL, PRs shouldn't publish anything